nectar/downloaders/threaded.py

```diff
@@ -97,6 +90,17 @@ def __init__(self, config, event_listener=None, tries=DEFAULT_TRIES, session=Non
 
         self.session = session or build_session(config)
 
+        # Configure an adapter to retry failed connections. Failed connection attempts are
```

Comment:
Looking at the docs for the ``Retry`` object, it retries more than just failed connection attempts. After being successfully connected, a read error will also result in a retry. I suggest just removing the word "attempts" and adjusting accordingly.